spec/helpers/firm_helper_spec.rb

```diff
@@ -81,4 +81,18 @@
       it { is_expected.to eq(I18n.t('investment_size.no_minimum')) }
     end
   end
+
+  describe 'minimum_fixed_fee' do
+    context 'displays values in GBP' do
+      before { allow(firm).to receive(:minimum_fixed_fee).and_return(10) }
+
+      it { expect(helper.minimum_fixed_fee(firm)).to eq('£10') }
+    end
+
+    context 'displays values in whole GBP' do
```

Comment:
Silly question, but was this a requirement? What if the firm's minimum fixed fee is £19.99?